controller/userTypeController.js

```diff
@@ -1,7 +1,11 @@
 const UserType = require('../model/userTypeModel');
 const HttpStatus = require('http-status-codes');
 
-const getAll = (req, res) => {
+const create = (req, res) => {
+    const userType = new UserType(req.body);
```

Comment:
Is this it? The whole function? O.o

Reply:
No validation because function will not be used.

Reply:
Also, you were probably looking at an outdated commit